blocks/parser.js

```diff
@@ -37,16 +39,14 @@ export function getBlockAttributes( blockNode, blockSettings ) {
  * @return {Array}          Block list
  */
 export default function parse( content ) {
-	return grammarParse( content ).reduce( ( memo, blockNode ) => {
+	return grammarParse( content ).map( ( blockNode ) => {
 		const settings = getBlockSettings( blockNode.blockType );
```

Comment:
How does the parser know that the generic block even exists? Would this be a pre-registered block maybe?